inference-engine/thirdparty/clDNN/src/graph_optimizer/remove_redundant_reorders.cpp

```diff
@@ -363,6 +363,45 @@ void remove_redundant_reorders::run(program_impl& p) {
         p.remove_if_dangling(*node);
     }
 
+    // This pass removes reorder for Convolution b_fs_yx_fsv16 -> bfyx
+    if (enable_reorder_fusing) {
+        itr = p.get_processing_order().begin();
+        while (itr != p.get_processing_order().end()) {
+            auto& node_ptr = *itr++;
+            if (!node_ptr->is_type<reorder>() ||
+                !node_ptr->is_in_data_flow() ||
+                //node_ptr->get_users().size() != 1 ||
+                node_ptr->get_dependencies().size() != 1) {
+                continue;
+            }
+
+            auto& node = node_ptr->as<reorder>();
+            auto& input = node.input();
+
+            if (!(input.is_type<convolution>()) ||
+                !(input.get_output_layout().format == format::b_fs_yx_fsv16) ||
+                !(node.get_output_layout().format == format::bfyx))
+                continue;
+
+            //if (usr->as<convolution>().get_primitive()->groups != 1)
+            //    continue;
+
```

Comment:
I think we need this condition

Reply:
Or if you are not going to support grouped convolution to be fused, please enable the above condition.

Reply:
Okay, I will enable above condition.